chapter_deep-learning-computation/model-construction.md

```diff
@@ -405,13 +405,16 @@ class MySequential(nn.Module):
 class MySequential(tf.keras.Model):
     def __init__(self, *args):
         super().__init__()
-        self._modules = tf.keras.Sequential()
+        self.modules = []
         for block in args:
-            # Here, block is an instance of a Model subclass.
-            self._modules.add(block)
+            # Here, block is an instance of a
+            # tf.keras.layers.Layer subclass.
+            self.modules.append(block)
```

Comment:
i feel it's problematic, as these layers are not initialized properly. if you check http://preview.d2l.ai/d2l-en/PR-1029/chapter_deep-learning-computation/model-construction.html#a-custom-block, the outputs are almost 0, which are not normal

Reply:
Good catch. I think that's because I accidentally had a relu activation. I just fixed it.